### PR TITLE
Fixed Issue 21588 - Rating icons changed to .svg

### PR DESCRIPTION
**Pull request by prisas**
Remade the rating images to solve the issue #21588.

### CODE REVIEW

**Comment by LukasReschke** [comment]
@paurisa You seem to have changed the permissions on the old files instead of deleted them. Was this a mistake? :)

**Comment by paurisa** [comment]
@LukasReschke I left them in because I didn't know if they would be necessary. I can remove them if you want.

**Comment by paurisa** [comment]
This contribution is MIT licensed.

**Comment by MorrisJobke** [comment]
On my machine those SVGs look more blurry than the PNGs before. How did you create those? There seems to be something gone wrong :(

**Comment by paurisa** [comment]
I just took the individual starts (from /core/img/actions) and re-created each rating image using those individual .svg's. I used the same size that the .png have and exported them as .svg.
I then searched how to correctly insert .svg's in php and I found that the way to do it is to use the image tag instead of the img one.
In my screen they look better but I don't have a retina display.

**Comment by deHoeninger** [comment]
It looks like the new SVGs are only containing the original PNG images and are not re-created as vector graphic. so they will not scale. Is that intended?

**Comment by paurisa** [comment]
No, they are suppose to scale but I don't know if there's a particular way to render them in php so they scale depending on the display resolution. Will take another look at it.

**Comment by MorrisJobke** [comment]
@paurisa This has nothing to do with php. They are also blurry if you open the file directly

**Comment by nickvergessen** [comment]
Files executable bit changed, can't comment on this in the diff...

File: core/img/rating/s0.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="96" height="18" viewBox="0 0 96 18">
+  <defs>
+    <image id="image" width="18" height="18" xlink:href="data:img/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAAASCAMAAABhEH5lAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAAM1BMVEUAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACjBUbJAAAAEHRSTlMAMGAYgBBQSAh4QGhwWCAooeEIPQAAAAFiS0dEAIgFHUgAAAAJcEhZcwAALiMAAC4jAXilP3YAAABvSURBVBjTXZBbDoAgEAOhVGFB8f63VRCWx3yRIWmbNaZhrdkBduPIY1Mn6VcThJRVWX70ggggsXEBdzROuCD560qzSa5GTw5hLPg5te/p6lHluxprNSzp9LIo5/IRNEpiXS0a5u92hAOxqekq5fkC5gYDV1r1bxgAAAAASUVORK5CYII="/>
```
Comment (MorrisJobke) [inline core/img/rating/s0.svg]: Ah. Now I know why it is blurry. It's an base64 encoded PNG inside of the SVG. ;)

**Comment by paurisa** [comment]
Images redone with InkScape and they now scale correctly (I think Gimp didn't export .svg's correctly).
Please comment if it works now.

**Comment by MorrisJobke** [comment]
Tested and works nicely :+1:

**Comment by MorrisJobke** [comment]
@LukasReschke @nickvergessen @rullzer @owncloud/designers Please review this :)

**Comment by LukasReschke** [comment]
:+1:

File: settings/js/apps.js [hunk 2]
```diff
@@ -3,9 +3,9 @@
 Handlebars.registerHelper('score', function() {
 	if(this.score) {
 		var score = Math.round( this.score / 10 );
-		var imageName = 'rating/s' + score + '.png';
+		var imageName = 'rating/s' + score + '.svg';
 
-		return new Handlebars.SafeString('<img src="' + OC.imagePath('core', imageName) + '">');
+		return new Handlebars.SafeString('<image src="' + OC.imagePath('core', imageName) + '">');
```
Comment (DeepDiver1975) [inline settings/js/apps.js]: image :question:

Comment (paurisa) [inline settings/js/apps.js]: @DeepDiver1975 I searched for a way to render .svg's correctly and found this article in Mozilla Developer Network saying that it's best to render them using the image tag. Is this incorrect?
https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/SVG_Image_Tag

Comment (Henni) [inline settings/js/apps.js]: @paurisa the `<image>` tag is part of the svg spec and can be used **within** SVGs.
But there is no `<image>` tag in HTML. You should probably leave it as it is (``)

Comment (MorrisJobke) [inline settings/js/apps.js]: This is for an image inside an SVG, but not an SVG within an web page. But I tested it and it nevertheless worked. :confused: Maybe it's best to make this the img tag.

Comment (paurisa) [inline settings/js/apps.js]: Gonna update it. Thanks for the information!

**Comment by paurisa** [comment]
Fixed! Now it uses the img tag again.

**Comment by MorrisJobke** [comment]
Retested and still works like a charm :+1:

**Comment by DeepDiver1975** [comment]
merged as https://github.com/owncloud/core/pull/23371

**Pull request closed without merging**